rq/utils.py

```diff
@@ -132,12 +132,18 @@ def import_attribute(name):
     # dotted path is not the last-before-end word
     # E.g.: package_a.package_b.module_a.ClassA.my_static_method
     # Thus we remove the bits from the end of the name until we can import it
+    #
+    # Sometimes the failure during importing is due to a genuine coding error in the imported module
+    # In this case, the exception is logged as a warning for ease of debugging.
+    # The above logic will apply anyways regardless of the cause of the import error.
     while len(module_name_bits):
         try:
             module_name = '.'.join(module_name_bits)
             module = importlib.import_module(module_name)
             break
         except ImportError:
+            logging.warning("Import error for '%s'" % module_name)
+            logging.warning("Import error trace:", exc_info=True)
```

Comment:
I'm ok with logging this. Just wondering whether this should be logged as warning or error.

Reply:
Many thanks, I chose warning as it's closer to the original logic (no warning at all). But error is surely an option.